Source/Core/Core/HW/WiimoteEmu/WiimoteEmu.cpp

```diff
@@ -284,6 +390,39 @@ Wiimote::Wiimote(const unsigned int index) : m_index(index), ir_sin(0), ir_cos(1
   // i18n: Refers to a 3D axis (used when mapping motion controls)
   m_shake->controls.emplace_back(new ControllerEmu::Input(ControllerEmu::Translate, _trans("Z")));
 
+  groups.emplace_back(m_shake_soft = new ControllerEmu::Buttons(_trans("ShakeSoft")));
+  // i18n: Refers to a 3D axis (used when mapping motion controls)
```

Comment:
The `i18n` comment must be on the line immediately before the `_trans`. Right now it's two lines above, which won't work.

Reply:
> Are you saying that the comment is read by some process?

Yes, xgettext.

> I can refactor..

That's not necessary. You should be able to just move the comments one line down (putting the comment in the middle of the statement).

Reply:
Thanks for the info!  Just to be sure, if these aren't used in the UI and we remove the `_trans` macro, do I still need to update the comment?

Reply:
If you remove the `_trans` macro, you should remove the corresponding `i18n` comment too.